docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb

```diff
@@ -93,8 +93,8 @@
     "work_dir = Path.cwd() / project_name\n",
     "data_dir = work_dir / 'data'\n",
     "\n",
-    "stack_start = parse_date('2019-06-10')\n",
-    "stack_end = parse_date('2019-07-21')\n",
+    "stack_start = parse_date('2019-06-10 00:00:00+00:00')\n",
+    "stack_end = parse_date('2019-07-21 00:00:00+00:00')\n",
```

Comment:
I slightly prefer `Z` for the time zone, since that's the format `asf_search` returns:
```suggestion
    "stack_start = parse_date('2019-06-10 00:00:00Z')\n",
    "stack_end = parse_date('2019-07-21 00:00:00Z)\n",
```